.github/workflows/ci.yml

```diff
@@ -310,6 +310,7 @@ jobs:
       - uses: actions/download-artifact@v3
         with:
           name: coverage
+      # Keep in sync with codecov.yml
```

Comment:
Keep *what* here in sync with *what* specifically in codecov.yml? There is a matching comment in `codecov.yml` at `after_n_builds: 10` but it isn't quite clear what that refers to.

Reply:
Good point! Is the new version better?

Reply:
Yeah, that’s much better!